src/containers/Ledger/index.js

```diff
@@ -90,36 +90,78 @@ class Ledger extends Component {
     const previousIndex = LedgerIndex - 1
     const nextIndex = LedgerIndex + 1
     const date = new Date(data.close_time)
+    const transactions = data.transactions || []
+    let successfulTxCount = 0
+    let failedTxCount = 0
+    const transactionTypes = {}
+    const ledgerObjTypes = {}
+
+    transactions.forEach((tx) => {
+      if (tx.result === 'tesSUCCESS') {
+        successfulTxCount += 1
+      } else {
+        failedTxCount += 1
+      }
+
+      if (transactionTypes[tx.type] === undefined) {
+        transactionTypes[tx.type] = 0
+      } else {
+        transactionTypes[tx.type] += 1
+      }
+
+      if (ledgerObjTypes[tx.ledgerEntryType] === undefined) {
```

Comment:
I'll look into it

Reply:
This can be fairly simply done probably like this:

```
const ledgerObjectsByType = {};
for (const obj of ledgerState) {
  if (!ledgerObjectsByType[obj["LedgerEntryType"]]) {
    ledgerObjectsByType[obj["LedgerEntryType"]] = 0;
  }
  ledgerObjectsByType[obj["LedgerEntryType"]] += 1;
}
```

And also for the UI, key is the title (AccountRoot, DirectoryNode, RippleState), and value is number of this objects for this ledger.

Reply:
I recommend doing ledger objects in a separate PR, since that will require a separate microservice. You'll only get the first page of ledger objects this way, instead of all of them.

Reply:
> I recommend doing ledger objects in a separate PR, since that will require a separate microservice. You'll only get the first page of ledger objects this way, instead of all of them.

Makes sense, for now I will add transaction types in this PR